library.json

```diff
@@ -1,9 +1,10 @@
+
 {
   "title": "Course Presentation",
   "contentType": "Library",
   "majorVersion": 1,
-  "minorVersion": 16,
-  "patchVersion": 1,
+  "minorVersion": 17,
+  "patchVersion": 3,
```

Comment:
Sure, I bumped it during my commits as I patched a few fixes, thanks for taking a look!